### ISSUE TITLE
Saving democracy via SMS + email election reminders

### ISSUE THREAD

**Issue by debracleaver** [sfbrigade/project-ideas#2]
**Short description**
People miss elections all the time because they literally forget to A) register to vote or B) sign up for an absentee ballot.  Let's help them out by sending them SMS and email reminders that elections are coming.  

**Challenges:**
- pulling together a reliable list of upcoming elections nationally -- there are groups that keep this data online that likely have info we can download, although probably not via API
- matching folks with relevant elections, likely based on zip code. (Not super hard, but something that needs to happen)
- working backward from election dates so the messages are sent in time.  
- integrating with something like Twilio to send the SMS messages
- integrating with something like Mandrill to send the election reminders
- making sure we give folks a way to unsubscribe, so we don't get blocked as spammers
- giving folks a way to update their zip codes, so we're not sending them reminders for elections they're no longer eligible for if they move.

About me: I run longdistancevoter.org, one of the most heavily trafficked election websites.  We've got a decent sized database of email addresses and cell phone numbers, that grows by leaps and bounds.  We also work closely with groups like Rock the Vote and HeadCount.org, and will happily share anything we build with other groups so that everyone benefits.  Let's save democracy together. 

**Project Needs (dev/design/resources)**
- Developers.  Very little design needed

**Status (in progress, pie-in-the-sky)**
- Mostly in progress.  We'll be getting free credits from Twilio, we're already set up over at Mandrill, we have a giant email list.

**Comment by raphaelmerx** [comment]
This seems like a very good idea. I would be interested in working on it. Has anyone started coding on this project?

**Comment by gitsteph** [comment]
I hope this is moving forward!  Please keep us posted-- I'd be really interested in helping out if you need some devs.  :)

**Comment by debracleaver** [comment]
@gitsteph : yes! @raphaelmerx and i spoke last week and he's interested in moving this forward as well. the more the merrier is my take on this.  are you SF based?  can we grab coffee and/or chat via google hangouts?

**Comment by gitsteph** [comment]
I AM indeed SF based!!  This is super exciting.  Yes for sure, let's grab coffee sometime soon.  Would Friday AM work?  Is there a way to private message you here?  I'm going to look that up now, heh.

**Comment by raphaelmerx** [comment]
Hi @gitsteph, this is indeed moving forward. I started working on it this weekend: https://github.com/raphaelmerx/election-reminders. We should meet and talk about how we can work together on it. I'll email you.

**Comment by debracleaver** [comment]
@gitsteph : i'm debra @ long distance voter dot org.  :)

**Comment by gitsteph** [comment]
Hi Debra!  I was able to meet Raphael today-- he brought me up to speed on how far he's come already with the app and what's left to do.  I'm def still interested in meeting up to talk with you at some point though and will send you an email off-forum soon at the email address you specified.  :)
